utils/swift_build_support/swift_build_support/products/cmark.py

```diff
@@ -86,7 +86,7 @@ def should_test(self, host_target):
         if self.is_cross_compile_target(host_target):
             return False
 
-        return self.args.test
+        return self.args.test_cmark
```

Comment:
Is it expected that we are aligning the behaviour with the current `build-script` products instead of following the old one of also listening to the value of `--test` (as per https://github.com/apple/swift/blob/7fd69a04b4d9a558c6c2b019b4c3e7db987a44cc/utils/build-script#L656) ?

It seems reasonable, but asking to double check my understanding.

Reply:
If `--test` is true, the test will run, unless `--skip-test-cmark` is also true.

Reply:
Does this imply that not specifying `--test` will cause CMark tests to be skipped? Or that `--test` and `--skip-test-cmark` only act on their respective tests (Swift for the former, CMark for the latter)?

Reply:
Yes, not specifying `--test` means cmark tests will be skipped.